deployment/sriov-network-operator/templates/operator.yaml

```diff
@@ -47,6 +47,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: K8S_POD_NAME
```

Comment:
1. you already have POD_NAME defined below
2. you can use `spec.nodeName` to expose Node name directly and avoid logic added to get node name from pod

Reply:
removed `K8S_POD_NAME` and added `NODE_NAME`